ros/src/tl_detector/light_classification/simple_cnn_classifier.py

```diff
@@ -0,0 +1,83 @@
+import cv2
+import rospy
+import rospkg
+
```

Comment:
For consistency with other files, this file should be named as `simple_cnn_tl_classifier.py` instead of just `simple_cnn_classifier.py`.

Reply:
changed